### DISCUSSION TITLE
How can I get the debug info If project start failed

### DISCUSSION THREAD

**Discussion by JohnWickzzz** [ayamir/nvimdots#1103] [Q&A] [answered]
I found that if project start failed, the dapui will close automatically. After that, I can't get the error message.
Although can use the command "lua require("dapui").open" to reopen the dapui, but it's not very convenient.
Is there a better way to debug? Or this is the correct way to use it.

**Comment by charliie-dev** [comment, marked as answer]
https://github.com/ayamir/nvimdots/blob/8bcc4099aa17812abfcf35a9dea0e5673d461bb5/lua/keymap/tool.lua#L138C1-L199

**Comment by JohnWickzzz** [comment]
> https://github.com/ayamir/nvimdots/blob/8bcc4099aa17812abfcf35a9dea0e5673d461bb5/lua/keymap/tool.lua#L138C1-L199

Ok, thank you! I set the shortcut keys myself for dapui open&close
